HW02/Tatsiana_Zenkovich/tatsiana_zenkovich_hw_02_t_02.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+lines = []
+
+File.open('log.txt') do |review_file|
+  lines = review_file.readlines
+end
+
+re = []
+lines.each do |i|
+  ip = i[/\d+\.\d+\.\d+\.\d+/]
+  date = i[%r{\[\d+/\w+/\d{4}\:\d{2}\:\d{2}\:\d{2}\s\+\d+\]}]
```

Comment:
We can move these variables to CONSTANT
```suggestion
  IP = i[/\d+\.\d+\.\d+\.\d+/]
  DATE = i[%r{\[\d+/\w+/\d{4}\:\d{2}\:\d{2}\:\d{2}\s\+\d+\]}]
```

Reply:
Why do you suggest moving them to constant? what's wrong with leaving it as a variable?

Reply:
It's just best practice. If we aren't going to change variables, we should move them to constants

https://stackoverflow.com/questions/29927103/when-to-use-constants-instead-of-instance-variables-in-ruby

Reply:
I changed as suggested, and now get many warnings during a code execution like this. Should I just ignore it?
"tatsiana_zenkovich_hw_02_t_02.rb:9: warning: already initialized constant IP
tatsiana_zenkovich_hw_02_t_02.rb:9: warning: previous definition of IP was here
tatsiana_zenkovich_hw_02_t_02.rb:10: warning: already initialized constant DATE
tatsiana_zenkovich_hw_02_t_02.rb:10: warning: previous definition of DATE was here
tatsiana_zenkovich_hw_02_t_02.rb:11: warning: already initialized constant ADDRESS"

Reply:
despite those warnings, the code makes correct output

Reply:
You need to move these constants outside of `each` block, there's no sense to redefine them everytime